recipes-core/images/cvm-initramfs.bbappend

```diff
@@ -0,0 +1,2 @@
+PACKAGE_INSTALL:append = " rbuilder"
+IMAGE_FEATURES:append = "debug-tweaks"
```

Comment:
Is this needed?

Reply:
actually, I added this here but I was thinking to create a branch for the dev purposes.
This should not be there for the final production deployment. But this is needed for having ssh access for debugging purposes at the moment.
I can add a comment above it as TODO: remove when productionizing/finalizing the rbuilder + yocto image

Reply:
Could we maybe have a separate "debug" bbappend that's included or something?

Reply:
We could do something similar to the other PRs where we have the python script fetching if DEBUG was enabled and append this debug-tweaks to the image conditionally.
How does this sound?

Reply:
I experimented a bit with this and I think I can move this back to the meta-confidential layer and have there the debug value to be set from an env variable prior to building.
I managed to make that work